src/client.rs

```diff
@@ -731,6 +731,25 @@ impl Client {
         self._pull_image_manifest(image).await
     }
 
+    /// Pull a manifest from the remote OCI Distribution service without parsing it.
+    ///
+    /// The client will check if it's already been authenticated and if
+    /// not will attempt to do.
+    ///
+    /// A Tuple is returned containing the plain text representation of the manifest
+    /// and the manifest content digest hash.
+    pub async fn pull_manifest_raw(
+        &self,
+        image: &Reference,
+        auth: &RegistryAuth,
+        accepted_media_types: &[&str],
+    ) -> Result<(String, String)> {
```

Comment:
I still think I'd prefer to have the symmetry of this being a `Vec<u8>` because I think that is what would be expected when pulling something raw. I totally understand the motivation behind keeping it a string though. I think the validate function should just be updated to take `&[u8]`, but I don't want to make you do that if you don't want to. So if you'd like to do it, let me know. Otherwise, I can merge this after a day or so if I haven't heard from you and make the change in a follow up PR!

Reply:
I agree with @thomastaylor312 on this request

Reply:
Perfectly reasonable, I'll see what I can do!